roles/openshift_common/tasks/atomic_docker_check.yml

```diff
@@ -0,0 +1,16 @@
+---
+# Fail early if Atomic and old version of Docker
+- block:
+
+  - name: Determine available Docker
+    shell: "rpm -q --queryformat '---\ncurr_version: %{VERSION}\n' docker"
```

Comment:
It does look like it only works for variables, and the raw/endraw syntax doesn't seem to be respected in an inline template.

I was able to get the expected result with the following:
```
- hosts: localhost
  become: no
  gather_facts: no
  connection: local
  vars:
  tasks:
  - command: docker version --format "{{ version_format }}"
    register: result
    vars:
      version_format: !unsafe '{{json .Server}}'
  - debug: var=result
```
output:
```
23:43:03 [jdetiber:~/git/github/openshift] % ansible-playbook test.yml
 [WARNING]: provided hosts list is empty, only localhost is available


PLAY [localhost] ***************************************************************

TASK [command] *****************************************************************
changed: [localhost]

TASK [debug] *******************************************************************
ok: [localhost] => {
    "result": {
        "changed": true, 
        "cmd": [
            "docker", 
            "version", 
            "--format", 
            "{{json .Server}}"
        ], 
        "delta": "0:00:00.027524", 
        "end": "2017-02-24 23:43:04.034446", 
        "rc": 0, 
        "start": "2017-02-24 23:43:04.006922", 
        "stderr": "", 
        "stdout": "{\"Version\":\"1.12.6\",\"ApiVersion\":\"1.24\",\"GitCommit\":\"ae7d637/1.12.6\",\"GoVersion\":\"go1.7.4\",\"Os\":\"linux\",\"Arch\":\"amd64\",\"KernelVersion\":\"4.9.7-201.fc25.x86_64\",\"BuildTime\":\"Mon Jan 30 16:15:28 2017\",\"PkgVersion\":\"docker-common-1.12.6-6.gitae7d637.fc25.x86_64\"}", 
        "stdout_lines": [
            "{\"Version\":\"1.12.6\",\"ApiVersion\":\"1.24\",\"GitCommit\":\"ae7d637/1.12.6\",\"GoVersion\":\"go1.7.4\",\"Os\":\"linux\",\"Arch\":\"amd64\",\"KernelVersion\":\"4.9.7-201.fc25.x86_64\",\"BuildTime\":\"Mon Jan 30 16:15:28 2017\",\"PkgVersion\":\"docker-common-1.12.6-6.gitae7d637.fc25.x86_64\"}"
        ], 
        "warnings": []
    }
}

PLAY RECAP *********************************************************************
localhost                  : ok=2    changed=1    unreachable=0    failed=0   
```